proposals/raster_to_points.json

```diff
@@ -0,0 +1,81 @@
+{
+    "id": "raster_to_points",
+    "summary": "Sample random points from rasters",
+    "description": "The given raster data cubes is converted into a set of point samples per geometry given. The samples are always taken at the pixel center.",
+    "categories": [
+        "cubes",
+        "vector"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "name": "data",
+            "description": "The raster data cube to extract a sample from.",
+            "schema": {
+                "type": "object",
+                "subtype": "raster-cube"
+            }
+        },
+        {
+            "name": "geometries",
+            "description": "Geometries to extract samples for. If not given or `null`, the bounding box of the raster data cube is used as the Polygon to sample for.\n\nTo maximize interoperability, a nested `GeometryCollection` should be avoided. Furthermore, a `GeometryCollection` composed of a single type of geometries should be avoided in favour of the corresponding multi-part type (e.g. `MultiPolygon`).",
```

Comment:
This sounds a bit strange imo: Maybe rather "Input Geometries for sample extraction". Same for the next sentence I would substitute "as the Polygon to sample" with "as input polygon" or something similar

Reply:
the rest is as we've discussed and should be fine for most of the applications

Reply:
Alright, fixed.